### ISSUE TITLE
gas consumption is missing

### ISSUE THREAD

**Issue by pbranly** [ulic75/power-flow-card#88]
Please add gas consumption
philippe

**Comment by ulic75** [comment]
I don't have gas measurement.  Do you have a screenshot of how gas appears in the official Energy Distribution card?

**Comment by pbranly** [comment]
![image](https://user-images.githubusercontent.com/22018509/179295469-e9a73091-2643-4740-896c-7b49beb00767.png)
![image](https://user-images.githubusercontent.com/22018509/179295480-650403f6-5f94-4e1f-89dc-45b6da3c37c6.png)
![image](https://user-images.githubusercontent.com/22018509/179295508-68301779-a44a-4b3f-84ba-dec7ebd7605c.png)
![image](https://user-images.githubusercontent.com/22018509/179295545-984c4a5b-58d8-438e-8163-0c2e2f00ef02.png)

**Comment by pbranly** [comment]
But as you can see the official distribution does not add grid and gas 
This is an issue that they don't (yet) want to correct 
Phil

**Comment by ulic75** [comment]
When you say gas are you meaning a gasoline generator that produces electricity or do you mean something like propane or natural gas which produces heat?

**Comment by pbranly** [comment]
Hi
I mean natural gas which produces heat for central heating system and hot water for bathroom.
In the home assistant energy menu, it possible to enter either volume in m3 with a kWh/m3 coefficient or kWh.
This is working fine but as said in a previous message, the developer does not want to correct the fact that grid (electricity) and gas energy are not added 
Regards 
Philippe

**Comment by joeph84** [comment]
I would love to see gas in the overview as well. For me the screenshot looks like this.
The gas is read by the P1 smart meter and the gas is used for hot water and heating.
![image](https://user-images.githubusercontent.com/46064468/184909678-9d33c53d-ba5e-4604-8cae-6c0c5e7f7bf1.png)

**Comment by pbranly** [comment]
It is a pity that in your exemple gas is equal to 0 m3
With a value , you should see that the "home" total does not take gas in consideration 
This is for me a bug 
Phil

**Comment by joeph84** [comment]
It is just about completeness of usage of utilities in the house. I would love to be able to add waterflow there as well.

**Comment by burrellka** [comment]
love it! great suggestion. and I second everyone here in saying this is an amazing card. Great job. 

I second what was suggested here. Add water two. To me this is the same as the other request to add two custom entities. So they done "add" to the total of the house, but are more informational based. Could be used for gas, water, a custom entity, etc. 

also, if the unit of measurements could be pulled from the entities (such as kW, kWh, etc) that would be awesome!

**Comment by ForumHulp** [comment]
Missing also gas and water, please add

**Comment by burrellka** [comment]
Hi @ulic75  , I just wanted to say thank you!!!! I  noticed you created a new branch for gas and water, and I went ahead and pulled you're latest pull request, compiled it and deployed it to my instance. I am absolutely loving the WIP!!!! I know you are super busy, and from me and the community I just wanted to say thank you. Can't wait until you have a chance to finish this and check the final code in 

Here is what it looks like on my instance

![image](https://user-images.githubusercontent.com/18409365/201693660-27aaca58-c307-4586-8eea-9aafd4e47418.png)

**Comment by burrellka** [comment]
Hi Hi @ulic75 , I saw that you made another commit for this that fixed the misaligned lines above. I pulled it down and deployed it. Thank you!!!! that fixed my issue (see below) . I am using to show today's consumption (I know no kwh, but that's minor and i'm good with it) and current consumption with no battery. Looks great!!! My only issue now is it only shows on my pc and the new code does not seem to render on mobile. But that's not a problem, I figured I just need to wait for the official release update in HACS for it to show in mobile. Any idea when you plan to release this to HACS? Thanks again for all your hard work, we all know you are busy and we appreciate it.


![image](https://user-images.githubusercontent.com/18409365/202171645-51a4d894-fa41-42d0-8353-97ed992f79eb.png)

![image](https://user-images.githubusercontent.com/18409365/202171681-a2c85140-de03-4b5f-89dd-6815ff0ab7f8.png)
